### PR TITLE
Add StashCache tests to param file

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

File: parameters.d/osg34-stashcache.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  # rng - Install entropy generation package (default: False)
+  ##################
+  - label: All + StashCache
+    osg_java: False
+    packages:
+      - osg-tested-internal
+      - stashcache-client
+      - stashcache-origin-server
+      - stashcache-cache-server
+      - stashcache-cache-server-auth
```
Comment (brianhlin) [inline parameters.d/osg34-stashcache.yaml]: Why do we need to test the case with `osg-tested-internal`? Are we planning on supporting other services installed alongside stashcache?

File: parameters.d/osg34-stashcache.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+sources:
+  ###################
+  # Format:
+  # [<Github account>:<osg-test branch>;] <OSG ver>; <REPO 1, REPO 2...REPO N> [> <Update OSG ver>/<Update REPO 1, REPO 2...REPO N>]
+  # Example:
+  # Run osg-test with packages from 3.2-release and 3.2-testing that are then upgraded to 3.3-testing and 3-3-upcoming-testing:
+  # 3.2; osg, osg-testing > 3.3/osg-testing, osg-upcoming-testing
+  ###################
+  # - opensciencegrid:master; 3.4; osg
```
Comment (brianhlin) [inline parameters.d/osg34-stashcache.yaml]: Why exclude the release repo?

File: parameters.d/osg34-stashcache.yaml (hunk 1 above)
Comment (matyasselmeci) [reply parameters.d/osg34-stashcache.yaml]: Well, I didn't notice that the tests + xrootd needed fixing until I installed both side-by-side.

File: parameters.d/osg34-stashcache.yaml (hunk 2 above)
Comment (matyasselmeci) [reply parameters.d/osg34-stashcache.yaml]: Not sure the version of SC in release works with the tests, but let me give it a shot...

File: parameters.d/osg34-stashcache.yaml (hunk 1 above)
Comment (matyasselmeci) [reply parameters.d/osg34-stashcache.yaml]: They collided by both running on the same port.

Comment (brianhlin) [reply parameters.d/osg34-stashcache.yaml]: That goes back to my point: do we expect to support a standalone XRootD installation alongside an XCache? I'm not sure why anyone would run that way

Comment (matyasselmeci) [reply parameters.d/osg34-stashcache.yaml]: Well, we don't expect anyone to install everything in osg-tested-internal either yet we still install it all in the nightlies. I'm hoping to eventually put SC into osg-tested-internal anyway.

(Did you know that at one point it was possible to have an osg-test run with zero skips? We even have code to mark a "perfect" run. Guess what's never going to happen again...)

Comment (matyasselmeci) [reply parameters.d/osg34-stashcache.yaml]: It's not a biggie though. If you don't want it in there, I can take it out.

File: parameters.d/osg34-stashcache.yaml (hunk 2 above)
Comment (matyasselmeci) [reply parameters.d/osg34-stashcache.yaml]: Looks like it works so I added it.

**Comment by matyasselmeci** [comment]
I think this should address all your concerns?

**Pull request merged**